Makefile

```diff
@@ -280,7 +280,10 @@ lint: docker-tools
 	@$(call print, "Linting source.")
 	$(DOCKER_TOOLS) golangci-lint run -v $(LINT_WORKERS)
 
-#? tidy-module: Run `go mod` tidy for all modules
+protolint:
```

Comment:
Looks like a comment got removed here during the rebase. Also, would be nice to also add a help comment (with the `#?` prefix) like the other make goals got in a recent PR.